tests/src/mocks.ts

```diff
@@ -245,18 +245,101 @@ export class KlerosLiquid {
     static bind(contract: Address) {
         return {
             courts: (id: BigInt2) => {
+                const courtsConfig = {
+                    0: {
+                        parent: new BigInt2(0),
```

Comment:
The court 0 hasn't a parent court in the graph.
In the SC has the court 0 as parent. Method 8 in the [kleros liquid smart contract](https://etherscan.io/address/0x988b3a538b618c7a603e1c11ab82cd16dbe28069#readContract)

what should be the correct behavior?

Reply:
It's working ok because of this check https://github.com/salgozino/kleros-subgraph/blob/master/src/KlerosLiquidMappings.ts#L482 , so the court 0 ends with parent=null instead of parent=0